packages/runtime/src/events/EventProxy.ts

```diff
@@ -125,7 +126,9 @@ export class EventProxy {
             this.targetEventBus.publish(new PeerSharedAttributeDeletedByPeerEvent(event.eventTargetAddress, AttributeMapper.toAttributeDTO(event.data)));
         });
 
-        this.subscribeToSourceEvent(consumption.ThirdPartyOwnedRelationshipAttributeDeletedByPeerEvent, (event) => {
+        this.subscribeToSourceEvent(consumption.ThirdPartyRelationshipAttributeDeletedByPeerEvent, (event) => {
+            this.targetEventBus.publish(new ThirdPartyRelationshipAttributeDeletedByPeerEvent(event.eventTargetAddress, AttributeMapper.toAttributeDTO(event.data)));
+            // The ThirdPartyOwnedRelationshipAttributeDeletedByPeerEvent is deprecated and will be removed in the next major version.
```

Comment:
please simplify these comments and just use `// deprecated`

Reply:
-> or just remove that comment, we already have a @deprecated on the event and the compiler will shout when removed.